lib/tessel/commands.js

```diff
@@ -28,21 +53,15 @@ module.exports.moveFolder = function(source, destination) {
 module.exports.untarStdin = function(filepath) {
   return ['tar', '-x', '-C', filepath];
 };
-module.exports.runScript = function(filepath, relpath) {
-  return ['node', filepath + relpath];
-};
 module.exports.openStdinToFile = function(filepath) {
   return ['dd', 'of=' + filepath];
 };
 module.exports.appendStdinToFile = function(filepath) {
   return ['tee', '-a', filepath];
 };
-module.exports.setExecutablePermissions = function(filepath) {
+module.exports.chmod = function(permission, filepath) {
   return ['chmod', '+x', filepath];
```

Comment:
I know this is a WIP, but I'm just adding a note so you don't forget to use `permission` in the return array.